packages/node/test/helpers/mockSdkInit.ts

```diff
@@ -42,6 +43,30 @@ export function cleanupOtel(_provider?: BasicTracerProvider): void {
   propagation.disable();
 }
 
+export function getSpanProcessor(): SentrySpanProcessor | undefined {
+  const client = getClient<NodeClient>();
+  if (!client?.traceProvider) {
+    return undefined;
+  }
+
+  const provider = getProvider(client.traceProvider);
+  if (!provider) {
+    return undefined;
+  }
+
+  // Access the span processors from the provider via _activeSpanProcessor
+  // Casted as any because _activeSpanProcessor is marked as readonly
+  const multiSpanProcessor = (provider as any)._activeSpanProcessor as
```

Comment:
not sure, but sometimes it also works to do `provider['_activeSpanProcessor']` to make TS happy!